LICENSE

```diff
@@ -1,6 +1,6 @@
-Copyright (c) 2015 Justin Crites
+The MIT License
 
-MIT License
+Copyright (c) 2013-2016 Plaid Technologies, Inc. <support@plaid.com>
```

Comment:
2013? The [initial commit][1] was in 2015.


[1]: https://github.com/plaid/plaid-ruby/commit/2ee9df00626e5e9213fcfe34c3161ea5d2e3a013